.yarnrc.yml

```diff
@@ -1,3 +1,3 @@
-nodeLinker: node-modules
+compressionLevel: mixed
 
-yarnPath: .yarn/releases/yarn-3.6.1.cjs
+enableGlobalCache: false
```

Comment:
what's this for?

Reply:
We were using a local yarn file which did not match the yarn version from `package.json`
Apparently yarn `corepack` is the new way.

Reply:
i'd prefer to not change the yarnrc in this PR. 

AFAICT, the current implementation works fine with corepack

```
> corepack enable
> yarn
> yarn build:debug
```

Reply:
Do you want a separate PR for `corepack`?
The current implementation work fine due to this [file](https://github.com/pola-rs/nodejs-polars/blob/main/.yarn/releases/yarn-3.6.1.cjs) but that's out of sync with what's in `package.json`

Reply:
let's just go with this.

```yaml
compressionLevel: mixed
enableGlobalCache: false
nodeLinker: node-modules
```

the main thing I wanted to avoid was using yarn's linker as it makes things more difficult for contributors.